project-keeper/src/main/resources/templates/.github/workflows/release.yml

```diff
@@ -95,23 +95,7 @@ jobs:
       - name: Verify release preconditions
         id: verify-release
         run: |
-          #mvn --batch-mode com.exasol:project-keeper-maven-plugin:verify-release --projects .
```

Comment:
Hm. I wonder if I ever saw a PR containing these sample and dummy script fragment. 
Glad to see it replaced by a proper maven call, now. :slightly_smiling_face: